ql/src/java/org/apache/hadoop/hive/ql/security/authorization/PrivilegeSynchronizer.java

```diff
@@ -218,12 +219,12 @@ public void run() {
               try {
                 tbl = hiveClient.getTable(dbName, tblName);
                 for (FieldSchema fs : tbl.getPartitionKeys()) {
-                  addGrantPrivilegesToBag(policyProvider, grantColumnBag, HiveObjectType.COLUMN,
-                          dbName, tblName, fs.getName(), authorizer);
+                  addGrantPrivilegesToBag(policyProvider, grantColumnBag, HiveObjectType.COLUMN, dbName, tblName,
+                      fs.getName(), authorizer);
                 }
                 for (FieldSchema fs : tbl.getSd().getCols()) {
-                  addGrantPrivilegesToBag(policyProvider, grantColumnBag, HiveObjectType.COLUMN,
-                          dbName, tblName, fs.getName(), authorizer);
```

Comment:
There is some formatting issue because of which so much diff is shown. Please fix this.

Reply:
Ok, made the changes.